src/modules/apps/install.ts

```diff
@@ -24,11 +25,32 @@ const promptPolicies = async () => {
   return promptConfirm('Do you accept all the Terms?')
 }
 
+const hasLicenseFile = async (name: string, version: string) => {
+  const client = createRegistryClient()
+  try {
+    await client.getAppFile(name, version, '/public/metadata/licenses/en-US.md')
+    return true
+  } catch (err) {
+    if (err.response?.status === 404) {
+      return false
+    }
+    throw err
+  }
+}
+
+const licenseURL = async (app: string, termsURL?: string): Promise<string | undefined> => {
+  const [name, argVersion] = app.split('@')
+  const version = argVersion ?? 'x'
+  return (await hasLicenseFile(name, version)) ? `https://apps.vtex.com/_v/terms/${name}@${version}` : termsURL
+}
+
 const checkBillingOptions = async (app: string, billingOptions: BillingOptions, force: boolean) => {
+  const { termsURL, type, free } = billingOptions
+  const license = await licenseURL(app, termsURL)
   log.warn(
     `${chalk.blue(app)} is a ${
-      billingOptions.free ? chalk.green('free') : chalk.red('paid')
-    } app. To install it, you need to accept the following Terms:\n\n${optionsFormatter(billingOptions)}\n`
+      isFreeApp(type, free) ? chalk.green('free') : chalk.red('paid')
```

Comment:
Other idea is to have a `Messages.ts` file like [Workspace Command](https://github.com/vtex/cli-plugin-workspace/blob/master/src/lib/constants/Messages.ts), in order to separate messages from code and group all of them.

Reply:
Very elegant idea, thanks for sharing! Done in 1a1f01e7e948ad1dcbe61c6a7cc761f9a28c6dd4